addOns/custompayloads/src/main/java/org/zaproxy/zap/extension/custompayloads/CustomPayloadsMultipleOptionsTablePanel.java

```diff
@@ -64,6 +74,7 @@ public CustomPayloadsMultipleOptionsTablePanel(
         addMissingDefaultsButton();
         addResetButton();
         addResetIdButton();
+        addPayloadFileButton();
```

Comment:
```suggestion
        addButtonSpacer();
        addPayloadFileButton();
```

As @thc202 previously mentioned :wink: